examples/kubernetes-config/kubernetes-clustersubnet.json

```diff
@@ -14,7 +14,7 @@
     },
     "agentPoolProfiles": [
       {
-        "name": "agent",
+        "name": "agentpool1",
```

Comment:
I'll revert them to "agent". Do you need all examples in kubernetes-config directory to use "agent" as pool name?